Sources/MapboxDirections/MapboxStreetClass.swift

```diff
@@ -1,65 +1,68 @@
     /// Important roads that are not motorways.
-    case Trunk = "trunk"
+    case trunk = "trunk"
     /// Link roads/lanes/ramps connecting to trunk roads
-    case TrunkLink = "trunk_link"
+    case trunkLink = "trunk_link"
     /// A major highway linking large towns.
-    case Primary = "primary"
+    case primary = "primary"
     /// Link roads/lanes connecting to primary roads
-    case PrimaryLink = "primary_link"
+    case primaryLink = "primary_link"
     /// A highway linking large towns.
-    case Secondary = "secondary"
+    case secondary = "secondary"
     /// Link roads/lanes connecting to secondary roads
-    case SecondaryLink = "secondary_link"
+    case secondaryLink = "secondary_link"
     /// A road linking small settlements, or the local centres of a large town or city.
-    case Tertiary = "tertiary"
+    case tertiary = "tertiary"
     /// Link roads/lanes connecting to tertiary roads
-    case TertiaryLink = "tertiary_link"
+    case tertiaryLink = "tertiary_link"
     /// Standard unclassified, residential, road, and living_street road types
-    case Street = "street"
+    case street = "street"
     /// Streets that may have limited or no access for motor vehicles.
-    case StreetLimited = "street_limited"
+    case streetLimited = "street_limited"
     /// Includes pedestrian streets, plazas, and public transportation platforms.
-    case Pedestrian = "pedestrian"
+    case pedestrian = "pedestrian"
     /// Includes motor roads under construction (but not service roads, paths, etc.
-    case Construction = "construction"
+    case construction = "construction"
     /// Roads mostly for agricultural and forestry use etc.
-    case Track = "track"
+    case track = "track"
     /// Access roads, alleys, agricultural tracks, and other services roads. Also includes parking lot aisles, public & private driveways.
-    case Service = "service"
+    case service = "service"
     /// Those that serves automobiles and no or unspecified automobile service.
-    case Ferry = "ferry"
+    case ferry = "ferry"
     /// Foot paths, cycle paths, ski trails.
-    case Path = "path"
+    case path = "path"
     /// Railways, including mainline, commuter rail, and rapid transit.
-    case MajorRail = "major_rail"
+    case majorRail = "major_rail"
     /// Includes light rail & tram lines.
-    case MinorRail = "minor_rail"
+    case minorRail = "minor_rail"
     /// Yard and service railways.
-    case ServiceRail = "service_rail"
+    case serviceRail = "service_rail"
     /// Ski lifts, gondolas, and other types of aerialway.
-    case Aerialway = "aerialway"
+    case aerialway = "aerialway"
     /// The approximate centerline of a golf course hole
-    case Golf = "golf"
+    case golf = "golf"
     /// Circular continuous-flow intersection
-    case Roundabout = "roundabout"
+    case roundabout = "roundabout"
     /// Smaller variation of a roundabout with no center island or obstacle
```

Comment:
A mini roundabout is also a point feature; the Streets source documentation doesn’t say so but that’s an omission.